fastfield_codecs/src/blockwise_linear.rs

```diff
@@ -300,13 +300,9 @@ impl FastFieldCodec for BlockwiseLinearCodec {
         // If this doesn't overflow the algorithm should be fine
         let theorethical_maximum_offset =
             fastfield_accessor.max_value() - fastfield_accessor.min_value();
-        if fastfield_accessor
+        fastfield_accessor
             .max_value()
-            .checked_add(theorethical_maximum_offset)
-            .is_none()
-        {
-            return None;
-        }
+            .checked_add(theorethical_maximum_offset)?;
```

Comment:
I prefer the earlier version... By far actually.

In the new version the intent is hidden.

Reply:
it's from a clippt lint, I also think the previous version is more clear. We could use [overflowing_add](https://doc.rust-lang.org/std/primitive.i8.html#method.overflowing_add) instead and check the bool, or disable the lint

Reply:
What about disabling the lint locally?